#! /usr/bin/env python

PACKAGE='raw_perception_tests'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

#        Name                              Type         Description                   Default Min    Max
gen.add("point_min_height",                double_t, 0, "Point min height",           0.005,  0.0,   0.05 )
gen.add("point_max_height",                double_t, 0, "Point max height",           3,      0.0,   5    )
gen.add("object_min_height",               double_t, 0, "Object min height",          0.01,   0.0,   0.5  )
gen.add("object_cluster_tolerance",        double_t, 0, "Object cluster tolerance",   0.01,   0.001, 0.3  )
gen.add("object_cluster_min_size",         int_t,    0, "Object cluster min size",    15,     5,     500  )
gen.add("object_cluster_max_size",         int_t,    0, "Object cluster max size",    5000,   100,   30000)

exit(gen.generate(PACKAGE, "tabletop_cluster_extractor_test", "TabletopClusterExtractorTest"))
